src/posts/action.md

```diff
@@ -1,30 +1,35 @@
-- **Sign petitions**: [International AI Treaty](https://aitreaty.org), [Ban Superintelligence](https://chng.it/Djjfj2Gmpk), [Demand responsible AI](https://www.change.org/p/artificial-intelligence-time-is-running-out-for-responsible-ai-development-91f0a02c-130a-46e1-9e55-70d6b274f4df), or one of the **national petitions**: [UK](https://petition.parliament.uk/petitions/639956), [AUS](https://www.aph.gov.au/e-petitions/petition/EN5163), [NL](https://aipetitie.nl)
+- **Write an email to your politicians** - we've found emails are surprisingly effective and take relatively little effort. Try to convince your government to work towards a pause or enforce other binding AI safety legislation. If you don't know what to write, [start with our email builder](/email-builder). When you get a meeting, you may want to use the following [lobby tips](/lobby-tips). (If you're in the US, [Democracy.io](https://democracy.io) makes it easier to send emails to all your federal representatives.)
+- **Call your politicians** - If you're okay talking on the phone, try calling legislators' offices. You may want to have a set of talking points in view so you stay on topic.
+- **Show up to politicians' offices** - Many legislators have offices where members of the public can visit. The most effective.
+- **Join / Organize a Demonstration**: Joining an in-person gathering, even a small group, magnifies your impact. See part 3 for more information on joining / organizing protests with PauseAI.
+- **Sign petitions**: [International AI Treaty](https://aitreaty.org), [Ban Superintelligence](https://chng.it/Djjfj2Gmpk), [Demand responsible AI](https://www.change.org/p/artificial-intelligence-time-is-running-out-for-responsible-ai-development-91f0a02c-130a-46e1-9e55-70d6b274f4df), or one of the **national petitions**: [UK](https://petition.parliament.uk/petitions/639956), [AUS](https://www.aph.gov.au/e-petitions/petition/EN5163), [NL](https://aipetitie.nl).
 
-### 2. Inform others
+### 2. Inform people around you
 
-- **Share** about AI risk on your social media. Tag @pauseai (Twitter), or @pause_ai (Instagram) in your posts.
+- **Share** about AI risk on your social media. Tag @pauseai (Twitter), @pause_ai (Instagram) in your posts. If you want to use our social media posts, send us a message on social media or [the Discord](https://discord.gg/2XXWXvErfA).
 - **Talk** to people in your life about AI safety. Answer their questions, and encourage them to act too. Use our [counterarguments](/counterarguments) to help you be more persuasive.
+- **Meet with local groups** - many cities have (free / low-cost) events about AI & technology policy. Attending these events is a great way to network and share your concerns. If you want AI safety marketing materials, reach out to us on [Discord](https://discord.gg/2XXWXvErfA) so we can send you some.
```

Comment:
Why did you delete

> - [**Tabling**](/tabling) and [**flyering**](/flyering) are great ways to reach many people in a short amount of time.

I mention it in the other PR but maybe it was not clear what I was referring to:

>  I think they are useful and I'm pretty sure it is almost the only place where they are linked